TIL/topics/dart_instance_class/instance_class.md

```
@@ -0,0 +1,155 @@
+
+- 오브젝트를 가상 세계 용으로 구체화 한 것 (붕어빵 틀)
+
+## 인스턴스 (instance)
+
+- 클래스를 활용 해 메모리 상에 만들어 낸 것 (붕어빵)
+
+## Dart에서 클래스 구현
+
+```dart
+class Hero {
+	String name;
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)